main.js

```diff
@@ -15,17 +18,31 @@ var activityError = document.querySelector(".description-error");
 var minutesError = document.querySelector(".minutes-error");
 var secondsError = document.querySelector(".seconds-error");
 
-studyBtn.addEventListener("click", activateStudyColor);
-meditateBtn.addEventListener("click", activateMeditateColor);
-exerciseBtn.addEventListener("click", activateExerciseColor);
+var currentActivity = {};
+
+var activityCategory = '';
+
+studyBtn.addEventListener("click", activateStudy);
+meditateBtn.addEventListener("click", activateMeditate);
```

Comment:
Dig that these handlers are going to changing the colors of the buttons that are clicked and assigning the category to the instance that's being created.